sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -2,6 +2,29 @@
 
 ## 7.0.0b5 (Unreleased)
 
+**New Features**
+
+* Added new properties to Message, PeekMessage and ReceivedMessage: `content_type`, `correlation_id`, `label`,
+`message_id`, `reply_to`, `reply_to_session_id` and `to`. Please refer to the docstring for further information.
+
+* Add new properties to PeekedMessaged and ReceivedMessage: `enqueued_sequence_number`, `dead_letter_error_description`,
+`dead_letter_reason`, `dead_letter_source`, `delivery_count` and `expires_at_utc`. Please refer to the docstring for further information.
+
+**Breaking Changes**
+
+* Removed/Renamed several properties and instance variables on Message (the changes applied to the inherited Message type PeekMessage and ReceivedMessage).
+  - Renamed property `user_properties` to `properties`
+      - The original instance variable `properties` which represents the AMQP properties now becomes an internal instance variable `_amqp_properties`.
+  - Removed property `enqueue_sequence_number`.
+  - Removed property `annotations`.
+  - Removed instance variable `header`.
+
+* Removed several properties and instance variables on PeekMessage and ReceivedMessage.
+  - Removed proeprty `partition_id` on both type.
+  - Removed property `settled` on both type.
```

Comment:
I'm slightly surprised we're yanking this.  I can think of reasonable situations in which it'd be useful.

Reply:
aha, I basically removed everything that C# doesn't have.

But your question makes me revisit the property on four languages:
No for Java and .Net, Yes for JS -- they have `isSettled`.

I would bring the topic into the SB sync up. thanks for pointing out.

Reply:
added it back for ReceivedMessage first (but still removed it from PeekMessage as I found it's meaningless to a PeekMessage).

As there is pending discussion on this property, we could create another PR if we decided to remove it in the future -- not blocking this one.